inc/Engine/Media/AboveTheFold/Frontend/Controller.php

```diff
@@ -148,7 +148,7 @@ private function set_fetchpriority( $lcp, string $html ): string {
 			'#<img(?:[^>]*?\s+src=["\']' . $url . '["\'][^>]*?|[^>]*?)>#',
 			function ( $matches ) {
 				// Check if the fetchpriority attribute already exists.
-				if ( preg_match( '/fetchpriority=[\'"]([^\'"]+)[\'"]/', $matches[0] ) ) {
+				if ( preg_match( '/fetchpriority\s*=\s*[\'"]([^\'"]+)[\'"]/', $matches[0] ) ) {
```

Comment:
Do we need to handle if the `fetchpriority` is in capital letters like:
https://regex101.com/r/tXEJJN/1